src/modules/admin/components/formRules/FormRule.tsx

```diff
@@ -51,69 +59,81 @@ const FormRule: React.FC<Props> = ({
-        label={`Funding Source is ${HmisEnums.FundingSource[funder]}`}
-      />
-    );
+  if (formRole === FormRole.Service) {
+    if (serviceType) {
+      conditions.serviceType = (
+        <FormRuleChip label={`Service is ${serviceType.name}`} />
+      );
+    } else if (serviceCategory) {
+      conditions.serviceCategory = (
+        <FormRuleChip label={`Service is ${serviceCategory.name}`} />
+      );
+    }
   }
 
+  if (!nonProjectFormRoles.includes(formRole)) {
+    if (projectType)
+      conditions.projectType = (
+        <FormRuleChip
+          label={`Project Type is ${HmisEnums.ProjectType[projectType]}`}
+        />
+      );
+    if (project)
+      conditions.project = (
+        <FormRuleChip label={`Project is ${project.projectName}`} />
+      );
+    if (organization)
+      conditions.organization = (
+        <FormRuleChip
+          label={`Organization is ${organization.organizationName}`}
+        />
+      );
+    if (otherFunder) {
+      conditions.otherFunder = (
+        <FormRuleChip label={`Funding Source is ${otherFunder}`} />
+      );
+    } else if (funder) {
+      conditions.funder = (
+        <FormRuleChip
+          label={`Funding Source is ${HmisEnums.FundingSource[funder]}`}
+        />
+      );
+    }
+  }
+
+  const nonClient = nonClientFormRoles.includes(formRole);
+
+  const conditionCount = Object.keys(conditions).length;
+
   return (
     <Stack direction='row' gap={2}>
       <Typography variant='body2' sx={{ flexGrow: 1 }}>
-        Applies to {appliesTo}
-        {conditions.length > 0 && (
+        {!nonClient && <>Applies to {appliesTo}</>}
+        {nonClient && conditionCount === 0 && (
           <>
-            {' '}
-            if{' '}
-            {conditions.map((condition, i) => {
+            Applies to <FormRuleChip label='All Projects' />
+          </>
+        )}
+        {conditionCount > 0 && (
+          <>
+            {nonClient ? 'If ' : ' if '}
+            {Object.entries(conditions).map(([key, condition], i) => {
               return (
-                <span key={condition.key}>
+                <span key={key}>
                   {condition}
-                  {i < conditions.length - 1 && ' and '}
+                  {i < conditionCount - 1 && ' and '}
                 </span>
               );
             })}
           </>
         )}
       </Typography>
-      <ActiveChip active={active} />
-      <ButtonTooltipContainer title={rule.system ? 'System rule' : undefined}>
-        <IconButton
-          aria-label='edit form rule'
-          disabled={rule.system}
-          onClick={() => {
-            setSelectedRule(rule);
-            openFormDialog();
-          }}
-          size='small'
-        >
-          <EditIcon fontSize='inherit' />
-        </IconButton>
-      </ButtonTooltipContainer>
+      <ActiveChip
+        active={active || false}
+        // TODO(#6289) - Deletion should deactivate. Inactive rules should not be shown/returned
```

Comment:
On this branch, I'm not seeing a way to edit/deactive/delete existing rules. That is a functionality regression, so we should either add the deletion button on this branch, or keep the old "edit" functionality until it's ready.